spec/unit/mutest/cli_spec.rb

```diff
@@ -151,6 +151,19 @@
         let(:expected_integration) { Mutest::Integration::Rspec }
       end
 
+      context 'when specifying the default null integration explicitely' do
+        let(:flags) { %w[--use null] }
+        let(:expected_integration) { Mutest::Integration::Null }
+
+        before do
+          expect(Kernel).to receive(:require)
+            .with('mutest/integration/null')
+            .and_call_original
```

Comment:
That was in the other spec, the one for `#setup`, which is the 'one test that tests the process' part.

So, this test is the 'one test that checks existence of the file for all supported integrations' part.